test/clj/rems/api/test_user_settings.clj

```diff
@@ -16,6 +16,13 @@
   (let [user-id (str (UUID/randomUUID))]
     (test-helpers/create-user! {:eppn user-id})
 
+    (testing "default user settings without authentification"
+      (is (= {:status 401
+              :body "unauthorized"}
+             (-> (request :get "/api/user-settings")
+                 handler
+                 assert-unauthorized))))
```

Comment:
I think there is some confusion here. This test both checks the return value for `unauthorized` and calls a function called `assert-unauthorized`. I'm sure both should not be needed.

Reply:
hmm, I guess I should create something like ```read-unauthorized```

Reply:
Why does it need to do it? The fields are available in the response object as they are, aren't they? And you can check the status if you so wish. No helper required.